libs/src/sdk/api/tracks/types.ts

```diff
@@ -152,3 +152,26 @@ export const DeleteTrackSchema = z
   .strict()
 
 export type DeleteTrackRequest = z.input<typeof DeleteTrackSchema>
+
+export const SaveTrackSchema = z
+  .object({
+    userId: HashId,
+    trackId: HashId,
+    metadata: z.optional(
+      z.object({
+        isSaveOfRepost: z.boolean()
```

Comment:
should we expose this in the sdk? maybe we could add a jsdoc comment about what it does (though not sure if that's possible with zod schemas)

Reply:
Yeah let's expose it because we will need it in our client. And apparently you can add jsdoc comments to the schema! Just added